lib/rules/validate-jsdoc/enforce-existence.js

```diff
@@ -26,18 +27,27 @@ enforceExistence.configure = function(options) {
         'jsDoc.enforceExistence rule was not configured properly'
     );
 
+    var optionsToPolicyMap = [
```

Comment:
Also, please use 4 spaces as indent. Not sure why it wasn't reported by jscs. Bug?